internal/controller/conditions.go

```diff
@@ -89,14 +89,12 @@ var AllowedStringsToDisplayConditionErrorMessage = []string{
 }
 
 func (r *SecretSyncReconciler) updateStatusConditions(ctx context.Context, ss *secretsyncv1alpha1.SecretSync, oldConditionType string, newConditionType string, conditionReason string, shouldUpdateStatus bool) {
-	logger := log.FromContext(ctx)
-
 	if ss.Status.Conditions == nil {
 		ss.Status.Conditions = []metav1.Condition{}
 	}
 
 	if len(oldConditionType) > 0 {
-		logger.V(10).Info("Removing old condition", "oldConditionType", oldConditionType)
+		klog.V(10).InfoS("Removing old condition", "oldConditionType", oldConditionType)
```

Comment:
This is probably too high and will never get caught up in logs. Usually default log level is 2. Let's get this at V(4)